### PR TITLE
Hiding the extended file information for pretty printed code tab

### PR DESCRIPTION
**Pull request by Narrator**
Fixes Issue: #6733 

### Summary of Changes

* Added condition to check if source is being pretty printed and if so, hide the extended file information.

### Test Plan

- [x] Command-P opens the panel
- [x] Clicking “+” opens the panel
- [x] Clicking a source navigates to the source
- [x] Clicking the pretty print icon opens the pretty printed source tab
- [x] The extended file information is not shown in the pretty printed source tab's title.

### CODE REVIEW

**Comment by darkwing** [comment]
I think this may be a bit too broad; I think my intention was to hide the info *if the other tab is open*.  Thoughts?

**Comment by Narrator** [comment]
By other tab, do you mean the original minified counterpart of the pretty-printed tab? 

If that's the case, should the extended file information appear in the pretty-printed tab when someone closes the original source tab?

**Comment by darkwing** [comment]
*should the extended file information appear in the pretty-printed tab when someone closes the original source tab?* . Yep!  That seems reasonable to me!

File: yarn.lock [hunk 1]
```diff
@@ -3364,6 +3364,10 @@ devtools-config@^0.0.16:
   version "0.0.16"
   resolved "https://registry.yarnpkg.com/devtools-config/-/devtools-config-0.0.16.tgz#e7251788422f42a16aa1372b6adbebfcb7a74994"
 
+devtools-connection@0.0.12:
```
Comment (jasonLaster) [inline yarn.lock]: we shouldnt change the yarn.lock

**Comment by darkwing** [comment]
I'm going to close this for now as I think we misunderstand the idea and scope.

**Pull request closed without merging**